lambdas/indexer/.chalice/config.json.template.py

```diff
@@ -20,25 +25,22 @@
     "stages": {
         config.deployment_stage: {
             "lambda_functions": {
-                # FIXME: Brittle coupling between the string literal below and
-                #        the handler function name in app.py
-                #        https://github.com/DataBiosphere/azul/issues/1848
-                "contribute": {
+                indexer.app.lambda_name(indexer.contribute): {
```

Comment:
I would like us to consider one more variation:

```suggestion
                indexer.contribute.lambda_name: {
```

There may be adverse effects of referencing the app from each handler but I can't see any right now.